wp-content/themes/core/components/routes/search/Search_Controller.php

```diff
@@ -69,14 +71,47 @@ public function render_breadcrumbs(): void {
 	}
 
 	/**
-	 * @return Breadcrumb[]
+	 * @return array
 	 */
-	protected function get_breadcrumbs(): array {
-		$page = get_option( 'page_for_posts' );
-		$url  = $page ? get_permalink( $page ) : home_url();
+	public function get_search_form_args(): array {
+		$args = [
+			Search_Form_Controller::CLASSES => [ 'c-search--full', 'search-results__form' ],
+			Search_Form_Controller::FORM_ID => uniqid( 's-' ),
+			Search_Form_Controller::VALUE   => get_search_query(),
+		];
+
+		return $args;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_results_text_args() {
```

Comment:
Don't forget the return type: `public function get_results_text_args(): array {`,

Additionally it appears that the docblock return type is wrong.